### ISSUE TITLE
My var stays empty

### ISSUE THREAD

**Issue by huuub** [NicoLouis-be/home-assistant-bolt-energy#1]
I made an input_number and tried it. with
service: pyscript.get_bolt_data
data: 
  entity_id: input_number.gasrate
  subscription: online
  residential: res
  utility: gas

However the var was still 0

**Comment by NicoLouis-be** [comment]
Yes indeed, i noticed the same, bolt energy isn't consistent in creating their pdf. I search for specific text before and after the value but they change the format every time.

I will need to search for a different solution.

**Comment by Chacsam** [comment]
Should I conclude that this doesn't work anymore?
Too bad, as it looked promising

**Comment by NicoLouis-be** [comment]
indeed , i should archive this.
bold some sort of encrypts there PDF so selecting parts of texts is not possible. is scrambles them around.
that makes it not possible to scrape it
